srml/support/procedural/src/storage/transformation.rs

```diff
@@ -600,15 +600,15 @@ fn create_and_impl_instance(
 	instance_prefix: &str,
 	ident: &Ident,
 	doc: &TokenStream2,
-	const_names: &[(Ident, String)],
+	const_names: &[(Ident, String, String)],
 	scrate: &TokenStream2,
 	instantiable: &Ident,
 	cratename: &Ident,
 ) -> TokenStream2 {
 	let mut const_impls = TokenStream2::new();
 
-	for (const_name, partial_const_value) in const_names {
-		let const_value = format!("{}{}", instance_prefix, partial_const_value);
+	for (const_name, const_value_prefix, const_value_suffix) in const_names {
```

Comment:
Conceptually this makes no difference or? 
Why did you make this change?

Reply:
this makes a difference: for head of linkedmap the implementation is now `"Instance0head of MyModule MyStorage"`   instead of `"head of Instance0MyModule MyStorage"`

Reply:
this test failed before this PR: https://github.com/paritytech/substrate/pull/3690/files#diff-c3e43d84ecf048969752108967ffa7c6L76

Reply:
Ahh, just read your comment that we currently don't use it.